tests/test_git_aliases.zunit

```diff
@@ -17,6 +17,7 @@
 @teardown {
     # Restore users git configuration
     mv ~/.gitconfig.bak ~/.gitconfig
+    git config --global --remove-section alias 2>/dev/null || true
```

Comment:
🤔 this doesnt make sense to me. You're going to be altering the users git config rather then the temporary git config generated for the tests.

What are you trying to do here?